api/orchestration_movers_test.go

```diff
@@ -203,3 +240,60 @@ func Test_stopTaskRun(t *testing.T) {
 		}
 	}
 }
+
+func TestGetRunListById(t *testing.T) {
+
+	run := &DatamoverRun{
+		BytesTransferred:         aws.Int64(100),
+		BytesWritten:             aws.Int64(100),
+		EstimatedBytesToTransfer: aws.Int64(100),
+		EstimatedFilesToTransfer: aws.Int64(100),
+		FilesTransferred:         aws.Int64(100),
+		Status:                   aws.String("RUNNING"),
+		Result:                   &datasync.TaskExecutionResultDetail{},
+	}
+
+	type input struct {
+		group string
+		name  string
+		id    string
+	}
+	type output struct {
+		isErr bool
+		run   *DatamoverRun
+	}
+	tests := []struct {
+		name  string
+		input input
+		exp   output
+	}{
+		{"group empty", input{"", "name1", "1234"}, output{true, nil}},
+		{"name empty", input{"group1", "", "1234"}, output{true, nil}},
+		{"valid test", input{"group1", "name1", "1234"}, output{false, run}},
+	}
+
+	o := &datasyncOrchestrator{
+		account:        "",
+		server:         &server{},
+		sp:             &sessionParams{},
+		datasyncClient: ydatasync.Datasync{Service: newMockDataSync(t, nil)},
+		rgClient: yresourcegroupstaggingapi.ResourceGroupsTaggingAPI{
+			Service: newMockRGClient(t, nil),
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ctx := context.Background()
+			got, err := o.datamoverRunDescribe(ctx, tt.input.group, tt.input.name, tt.input.id)
+			gotErr := err != nil
+			if tt.exp.isErr != gotErr {
+				t.Error(tt.name, " expected error but did not receive error", err)
+			}
+			if got != nil {
+				got.StartTime = nil
```

Comment:
and you should be able to get rid of this

Reply:
Done..